mbuild/lattice.py

```diff
@@ -20,8 +20,8 @@ def load_cif(file_or_path=None, wrap_coords=False):
         Wrap the lattice points back into the 0-1 acceptable coordinates.
     """
 
-    garnett = import_('Garnett')
-    pycifrw = import_('pycifrw')
+    garnett = import_('garnett')
+    pycifrw = import_('CifFile')
```

Comment:
pycifrw isn't used anywhere. Do we need to import it or can we do some sort of assertion with `has_pycifrw`. It's not a very big package/long import, so this isn't really a big deal.

Reply:
We technically are, since Garnett requires pycifrw to read cif files. I ran into an issue before where I tried to load cif files but couldn’t, I did not realize pycifrw is a soft dependency for Garnett.